tests/psalm-baseline.xml

```diff
@@ -17,14 +17,42 @@
     </UndefinedMethod>
   </file>
   <file src="lib/AppInfo/Application.php">
-    <UndefinedClass occurrences="1">
+    <UndefinedClass occurrences="2">
       <code>BeforeTemplateRenderedEvent</code>
+      <code>CircleDestroyedEvent</code>
     </UndefinedClass>
   </file>
   <file src="lib/Folder/FolderManager.php">
     <MismatchingDocblockReturnType occurrences="1">
       <code>array{id: mixed, mount_point: mixed, groups: array&lt;empty, empty&gt;|mixed, quota: int, size: int|mixed, acl: bool}|false</code>
     </MismatchingDocblockReturnType>
+    <UndefinedClass occurrences="15">
+      <code>CircleDestroyedEvent</code>
+      <code>CirclesQueryHelper</code>
+      <code>CircleProbe</code>
+      <code>CircleNotFoundException</code>
+      <code>$federatedUser = $circlesManager->getLocalFederatedUser($user->getUID());</code>
+      <code>$federatedUser = $circlesManager->getLocalFederatedUser($user->getUID());</code>
+      <code>$queryHelper = $circlesManager->getQueryHelper();</code>
+      <code>$queryHelper = $circlesManager->getQueryHelper();</code>
+      <code>$queryHelper = $circlesManager->getQueryHelper();</code>
+      <code>$circlesManager->startSuperSession();</code>
+      <code>@param CirclesQueryHelper|null $queryHelper</code>
+      <code>$circlesManager->startSuperSession();</code>
+      <code>$circlesManager->getCircle($groupId, $probe);</code>
+      <code>$circlesManager->stopSession();</code>
+      <code>@param CirclesManager|null $circlesManager</code>
+    </UndefinedClass>
```

Comment:
We should fix this instead of committing the new issues :)

Reply:
This is (mostly?) undefined bits from the circles app, so we would need to add stubs for it